packages/logger/src/index.ts

```diff
@@ -1,25 +1,21 @@
-const winston = require('winston');
-const { format } = require('logform');
+import winston from 'winston';
+import { format } from 'logform';
 
-const logger = winston.createLogger({
+export const logger = winston.createLogger({
   transports: [
     new winston.transports.Console({
-      name: 'console',
       level: 'info',
       format: format.combine(format.colorize(), format.simple()),
     }),
   ],
 });
 
-logger.addFileLogging = function (filename) {
+export function enableFileLogging(filename: string) {
```

Comment:
While we are editing this code, could we add `level` as an argument to this function? I would like to be able to log `debug` and higher to `/var/log/prairielearn.log` and `error` and higher to `/var/log/prairielearn_error.log`, so that we can ship the error log to CloudWatch.

Reply:
Interesting idea! I make this change in 7fe6ffc. I ultimately added an options object to mirror the underlying `winston.transports.File` constructor.